code/modules/projectiles/updated_projectiles/ammo_datums.dm

```diff
@@ -397,7 +397,7 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 /datum/ammo/bullet/rifle/incendiary
 	name = "incendiary rifle bullet"
 	hud_state = "rifle_fire"
-	damage_type = BURN
+	damage = 20
```

Comment:
Damage is defined twice.

Reply:
Not seeing it, can you give me a suggested change when you get a chance?